cmd/lncli/main.go

```diff
@@ -3,11 +3,27 @@ package main
 import (
 	"fmt"
 	"os"
+	"path/filepath"
+	"strings"
 
 	"github.com/lightningnetwork/lnd/lnrpc"
+	"github.com/roasbeef/btcutil"
 	"github.com/urfave/cli"
 
+	flags "github.com/btcsuite/go-flags"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+const (
+	defaultConfigFilename  = "lnd.conf"
+	defaultSSLCertFilename = "ssl.cert"
+)
+
+var (
+	lndHomeDir        = btcutil.AppDataDir("lnd", false)
```

Comment:
Ah, maybe you have no access to it since it is in the lnd/main package... Still, it would be nice to have it located in a place where it could be accessed both from lnd/main and from lncli.